src/devMch.c

```diff
@@ -201,77 +201,79 @@
+static long read_fru_stringin(struct stringinRecord* pstringin);
+static long stringin_fru_ioint_info(int cmd, struct stringinRecord* pstringin, IOSCANPVT* iopvt);
 
 /* global struct for devSup */
 typedef struct {
-        long            number;
-        DEVSUPFUN       report;
-        DEVSUPFUN       init;
-        DEVSUPFUN       init_record;
-        DEVSUPFUN       get_ioint_info;
-        DEVSUPFUN       read_write;
-        DEVSUPFUN       special_linconv;
+    long      number;
+    DEVSUPFUN report;
+    DEVSUPFUN init;
+    DEVSUPFUN init_record;
+    DEVSUPFUN get_ioint_info;
+    DEVSUPFUN read_write;
+    DEVSUPFUN special_linconv;
 } MCH_DEV_SUP_SET;
 
 /* Add reporting */
-MCH_DEV_SUP_SET devAiMch         = {6, NULL, NULL,              init_ai_record,           ai_ioint_info,           read_ai,           NULL};
-MCH_DEV_SUP_SET devBoMch         = {6, NULL, NULL,              init_bo_record,           NULL,                    write_bo,          NULL};
-MCH_DEV_SUP_SET devMbboMch       = {6, NULL, NULL,              init_mbbo_record,         NULL,                    write_mbbo,        NULL};
-MCH_DEV_SUP_SET devBiMch         = {6, NULL, init_bi,           init_bi_record,           bi_ioint_info,           read_bi,           NULL};
-MCH_DEV_SUP_SET devMbbiMch       = {6, NULL, init_mbbi,         init_mbbi_record,         mbbi_ioint_info,         read_mbbi,         NULL};
-MCH_DEV_SUP_SET devLonginMch     = {6, NULL, NULL,              init_longin_record,       NULL,                    read_longin,       NULL};
-MCH_DEV_SUP_SET devAiFru         = {6, NULL, init_fru_ai,       init_fru_ai_record,       ai_fru_ioint_info,       read_fru_ai,       NULL};
-MCH_DEV_SUP_SET devLongoutFru    = {6, NULL, NULL,              init_fru_longout_record,  NULL,                    write_fru_longout, NULL};
-MCH_DEV_SUP_SET devStringinFru   = {6, NULL, init_fru_stringin, init_fru_stringin_record, stringin_fru_ioint_info, read_fru_stringin, NULL};
+MCH_DEV_SUP_SET devAiMch       = {6, NULL, NULL, init_ai_record, ai_ioint_info, read_ai, NULL};
+MCH_DEV_SUP_SET devBoMch       = {6, NULL, NULL, init_bo_record, NULL, write_bo, NULL};
+MCH_DEV_SUP_SET devMbboMch     = {6, NULL, NULL, init_mbbo_record, NULL, write_mbbo, NULL};
+MCH_DEV_SUP_SET devBiMch       = {6, NULL, init_bi, init_bi_record, bi_ioint_info, read_bi, NULL};
+MCH_DEV_SUP_SET devMbbiMch     = {6, NULL, init_mbbi, init_mbbi_record, mbbi_ioint_info, read_mbbi, NULL};
+MCH_DEV_SUP_SET devLonginMch   = {6, NULL, NULL, init_longin_record, NULL, read_longin, NULL};
+MCH_DEV_SUP_SET devAiFru       = {6, NULL, init_fru_ai, init_fru_ai_record, ai_fru_ioint_info, read_fru_ai, NULL};
+MCH_DEV_SUP_SET devLongoutFru  = {6, NULL, NULL, init_fru_longout_record, NULL, write_fru_longout, NULL};
+MCH_DEV_SUP_SET devStringinFru = {
+    6, NULL, init_fru_stringin, init_fru_stringin_record, stringin_fru_ioint_info, read_fru_stringin, NULL};
```

Comment:
I think this is caused by the `ColumnLimit`. I'm thinking we should disable the `ColumnLimit` entirely by setting it to 0. @marciodo thoughts?

Reply:
The ColumLimit is useful, for example, in functions that have several parameters or parameters with several characters. In this case, clang-format will split the arguments into more lines, making the code easier to read.

But for the case above, it would be better if clang-format could do something like:
```
MCH_DEV_SUP_SET devStringinFru = {6, NULL, init_fru_stringin, init_fru_stringin_record,
                                                                   stringin_fru_ioint_info, read_fru_stringin, NULL};
```
respecting the column limit.

Reply:
Hi guys, I couldn't find a way to achieve this. Do you have any idea how to do it?

Reply:
Maybe this is not possible. I'll try a few things, although I'm not sure when.